mvnw.cmd

```diff
@@ -43,7 +43,7 @@ title %0
 @REM set %HOME% to equivalent of $HOME
 if "%HOME%" == "" (set "HOME=%HOMEDRIVE%%HOMEPATH%")
 
-@REM Execute a user defined script before this one
+@REM Execute a userAccount defined script before this one
```

Comment:
this file should be ignored, git should not push this

Reply:
I added mvnw.cmd to .gitignore